src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -320,7 +320,7 @@ static void auh_entry_store(TimestampTz auh_time,
     AUHEntryArray[inserted].top_level_node_id[1] = 0;
   }
 
-  ereport(LOG, (errmsg("%llu %llu", AUHEntryArray[inserted].top_level_node_id[0], AUHEntryArray[inserted].top_level_node_id[1])));
```

Comment:
I used this line to debug something, and it got pushed to yb_auh. However, this has been removed from the latest yb_auh